scim/resource_user_test.go

```diff
@@ -459,7 +459,7 @@ func TestCreateForceOverwriteCannotListUsers(t *testing.T) {
 		d := ResourceUser().TestResourceData()
 		d.Set("force", true)
 		err := createForceOverridesManuallyAddedUser(
-			fmt.Errorf("User with username me@example.com already exists."),
+			fmt.Errorf("User with email me@example.com already exists in this account"),
```

Comment:
Maybe move this format string into a constant, or create a function that will accept email & return string?

Reply:
refactored the error messages into a function